recordium/main.py

```diff
@@ -257,13 +257,14 @@ def _item_clicked(self, widget):
         row = widget.row()
 
         if column == self.check_col:
-            # click in the checkbox column strike out (or unstrike) the whole row
-            should_strikeout = widget.checkState() == QtCore.Qt.Checked
+            # click in the checkbox column disable or enable the whole row
+            disable = widget.checkState() == QtCore.Qt.Checked
             for column in range(len(self.titles) - 1):  # -1 to not strike out checkbox
                 item = self.item(row, column)
-                font = item.font()
-                font.setStrikeOut(should_strikeout)
-                item.setFont(font)
```

Comment:
I think that disabling is a good step, but not final.

Let's try this: disable AND strike out the texts, that would give the impression that the item will be gone for good.

Thanks for this experimentation!!